iothub/device/tests/Transport/Mqtt/MqttTransportHandlerTests.cs

```diff
@@ -379,7 +383,7 @@ public async Task MqttTransportHandlerSendTwinGetAsyncHappyPath()
             var twinReturned = await transport.SendTwinGetAsync(CancellationToken.None).ConfigureAwait(false);
```

Comment:
This can validate `JsonConvert.DeserializeObject()` working properly with date-formatted string containing trailing zeros.